src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -300,11 +393,11 @@
  * import static java.lang.Math.abs; // OK, alphabetical case sensitive ASCII order, 'P' &lt; 'a'
  * import static org.abego.treelayout.Configuration.AlignmentInLevel; // OK, alphabetical order
  *
+ * import java.util.Set; // violation, extra separation in import group before 'java.util.Set'
```

Comment:
same here, no need to copy class name

Reply:
done

Reply:
> Are you not on checkstyle gitter?

I am there, but I do not visit it frequently